var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -138,6 +138,18 @@ class Gaudi(CMakePackage):
     # The Intel VTune dependency is taken aside because it requires a license
     depends_on("intel-parallel-studio -mpi +vtune", when="+vtune")
 
+    # /usr/bin/env triggers SIP, but we still need the LD_LIBRARY_PATH for gaudirun.py
+    # hardcode the absolute path instead
+    @when(sys.platform == "darwin")
+    @run_before("cmake")
+    def fix_script_shebang(self):
+        filter_file(
```

Comment:
There's an ~~app~~ method for that, e.g.https://github.com/spack/spack/blob/e9d2732e0072d992736403f61d1f3fc8a09905f8/var/spack/repos/builtin/packages/parallel/package.py#L56